drivers/gles_common/rasterizer_canvas_batcher.h

```diff
@@ -2282,7 +2282,19 @@ PREAMBLE(bool)::prefill_joined_item(FillState &r_fill_state, int &r_command_star
 #ifdef GLES_OVER_GL
 				// anti aliasing not accelerated .. it is problematic because it requires a 2nd line drawn around the outside of each
 				// poly, which would require either a second list of indices or a second list of vertices for this step
+				bool use_legacy_path = false;
+
 				if (polygon->antialiased) {
+					// anti aliasing is also not support for software skinned meshes.
+					// we can't easily revert, so we force software skinned meshes to run through
+					// batching path with no AA.
+					if (bdata.settings_use_software_skinning && p_item->skeleton != RID()) {
+					} else {
+						use_legacy_path = true;
+					}
```

Comment:
```suggestion
					if (!(bdata.settings_use_software_skinning && p_item->skeleton != RID())) {
						use_legacy_path = true;
					}
```

Reply:
Haha yeah that probably is better. I sometimes end up doing it the longhand way because I find it easier to reason with, but I am nuts. :+1: :grin:

Reply:
Or if you want to be a cool oneliner kid:
```cpp
use_legacy_path = !(bdata.settings_use_software_skinning && p_item->skeleton != RID());
```
or
```cpp
use_legacy_path = !bdata.settings_use_software_skinning || p_item->skeleton == RID()